i18n/translations/en.json

```
@@ -449,7 +449,10 @@
 	"language.variables.empty": "No translations yet",
 
 	"language.variable.delete.confirm": "Do you really want to delete the variable for {key}?",
+	"language.variable.entries": "Entries",
+	"language.variable.entries.help": "This strings will be used for translations count. Each entry corresponds to a different case counter. For example: 0, 1, 2 and more",
 	"language.variable.key": "Key",
+	"language.variable.multiple": "Multiple",
```

```suggestion
	"language.variable.multiple": "Different strings based on count",
```

Not so happy with this either. But I think the `multiple` field needs more explanatory label and maybe also on/off texts.

---

Absolutely. I'm not happy with the labels and help texts either. That's why I mentioned it in the PR description. It needs to be more descriptive and clear.